apps/minifront/src/components/root-router.tsx

```diff
@@ -34,12 +34,10 @@ export const rootRouter = createHashRouter([
             index: true,
             loader: AssetsLoader,
             element: <AssetsTable />,
-            errorElement: <ErrorBoundary />,
```

Comment:
Adding error boundaries lower down in the route tree means that the error boundary will render _within_ that route's `<Outlet />`. I've removed these so that all errors bubble up to the top-level boundary.